src/traces/scattermapbox/defaults.js

```diff
@@ -66,7 +66,14 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     coerce('cluster.enabled', clusterEnabledDflt);
 
     if(subTypes.hasText(traceOut)) {
-        handleTextDefaults(traceIn, traceOut, layout, coerce, {noSelect: true});
+        handleTextDefaults(traceIn, traceOut, layout, coerce,
+            {noSelect: true,
+                font: {
+                    family: 'Open Sans Regular',
+                    size: layout.font.size,
+                    color: layout.font.color
+                }
```

Comment:
```suggestion
                font: layout.font
```

Reply:
I tried to not change this, but couldn't get it to work with the glyphs. If you make this change, you get:
![image](https://github.com/plotly/plotly.js/assets/49296311/23634cff-540d-4f0d-abc5-6e3f889d2cf0)

The reason that I found was because `fontstack` in the glyphs key ends up being the entire family default. I'm not entirely sure where that gets set. Also, the font needs to be one of the items in [openmaptiles/fonts](https://github.com/openmaptiles/fonts/blob/gh-pages/fontstacks.json?rgh-link-date=2023-06-27T21%3A59%3A52Z), otherwise, you get a 404.

Reply:
Thanks for the note.
Let's add a comment and a function to handle cases that `layout.font.family` is not found in that list.
```js
var supportedFonts = [
    'Metropolis Black Italic',
    'Metropolis Black',
    'Metropolis Bold Italic',
    'Metropolis Bold',
    'Metropolis Extra Bold Italic',
    'Metropolis Extra Bold',
    'Metropolis Extra Light Italic',
    'Metropolis Extra Light',
    'Metropolis Light Italic',
    'Metropolis Light',
    'Metropolis Medium Italic',
    'Metropolis Medium',
    'Metropolis Regular Italic',
    'Metropolis Regular',
    'Metropolis Semi Bold Italic',
    'Metropolis Semi Bold',
    'Metropolis Thin Italic',
    'Metropolis Thin',
    'Open Sans Bold Italic',
    'Open Sans Bold',
    'Open Sans Extra Bold Italic',
    'Open Sans Extra Bold',
    'Open Sans Italic',
    'Open Sans Light Italic',
    'Open Sans Light',
    'Open Sans Regular',
    'Open Sans Semibold Italic',
    'Open Sans Semibold',
    'Klokantech Noto Sans Bold',
    'Klokantech Noto Sans CJK Bold',
    'Klokantech Noto Sans CJK Regular',
    'Klokantech Noto Sans Italic',
    'Klokantech Noto Sans Regular'
];
```
then
```js
                font: {
                    family: supportedFonts.indexOf(layout.font.family) !== -1 ? layout.font.family : 'Open Sans Regular',
                    size: layout.font.size,
                    color: layout.font.color
                }

Reply:
Updated.